app/grants/models.py

```diff
@@ -590,6 +608,33 @@ def successful_contribution(self, tx_id):
         except ConversionRateNotFoundError as e:
             logger.info(e)
 
+        if self.num_tx_processed == self.num_tx_approved:
+            try:
+                converted_amount = (
```

Comment:
i think that, in order to DRY, `usdt_amount` should be its own function, like in https://github.com/gitcoinco/web/pull/3300